twisted/protocols/memcache.py

```diff
@@ -303,7 +309,8 @@ def cmd_CLIENT_ERROR(self, errText):
         """
         An invalid input as been sent.
         """
-        log.err("Invalid input: %s" % (errText,))
+        errText = nativeString(errText)
```

Comment:
This should `repr()` instead -- the err will be `b''` prefixed on Python 3, but that's fine, as it means binary garbage coming in won't cause this to fail.